src/NzbDrone.Core/Download/Clients/DownloadStation/DownloadStation.cs

```diff
@@ -0,0 +1,302 @@
+            {
+                _logger.Error(ex, ex.Message);
+                return Enumerable.Empty<DownloadClientItem>();
+            }
+
+            var items = new List<DownloadClientItem>();
+
+            foreach (var torrent in torrents)
+            {
+                var outputPath = new OsPath(torrent.Additional.Detail["destination"]);
+
+                if (Settings.TvDirectory.IsNotNullOrWhiteSpace())
+                {
+                    if (!Settings.TvDirectory.Equals(outputPath.FullPath))
+                        continue;
+                }
+                else if (Settings.TvCategory.IsNotNullOrWhiteSpace())
+                {
+                    var directories = outputPath.FullPath.Split('\\', '/');
+                    if (!directories.Contains(Settings.TvCategory))
+                        continue;
+                }
+
+                outputPath = _remotePathMappingService.RemapRemoteToLocal(Settings.Host, outputPath);
+
+                var item = new DownloadClientItem()
+                {
+                    Category = Settings.TvCategory,
+                    DownloadClient = Definition.Name,
+                    DownloadId = torrent.Id,
+                    OutputPath = GetOutputPath(outputPath, torrent),
+                    Title = torrent.Title,
+                    TotalSize = torrent.Size,
+                    RemainingSize = torrent.Size - Convert.ToInt64(torrent.Additional.Transfer["size_downloaded"]),                     
+                    Status = GetTorrentStatus(torrent.Status),
+                    Message = GetMessage(torrent)                    
+                };
+
+                items.Add(item);
+            }
+
+            return items;
+        }
+
+        private string GetMessage(DownloadStationTask torrent)
+        {
+            if (torrent.StatusExtra != null)
+            {
+                switch (torrent.Status)
+                {
+                    case DownloadStationTaskStatus.Error:
+                        return torrent.StatusExtra["error_detail"];
+                    case DownloadStationTaskStatus.Extracting:
+                        return torrent.StatusExtra["unzip_progress"];
+                }
+            }
+
+            return null;
+        }
+
+        private DownloadItemStatus GetTorrentStatus(DownloadStationTaskStatus status)
+        {
+            if (StatusTable.ContainsKey(status))
+            {
+                return StatusTable[status];
+            }
+
+            return DownloadItemStatus.Downloading;
+        }
+
+        public override DownloadClientStatus GetStatus()
+        {
+            var data = _proxy.GetConfig(Settings);
+            
+            try
+            {
+                var path = data["default_destination"] as string;
+
+                if (Settings.TvCategory.IsNotNullOrWhiteSpace())
+                {
+                    path = string.Format("{0}/.{1}", path, Settings.TvCategory);
```

Comment:
My comment was more toward markus.
replace it with `$"{path}/{Settings.TvCategory}";` for now and ignore what I said.